### ISSUE TITLE
Is the bot back up

### ISSUE THREAD

**Issue by Kours** [finndev/PokeBuddy#294]

**Comment by Ander0128** [comment]
I don't believe so, You can look at 218 and refer to this post to keep up on the updates.

https://www.reddit.com/r/pokemongodev/comments/4w1cvr/pokemongo_current_api_status/

**Comment by viktrok** [comment]
not yet :(

**Comment by dodolock** [comment]
ye the unknown6 ripped us all

**Comment by bertje123** [comment]
Released: https://github.com/keyphact/pgoapi
